src/MaiaJob.php

```diff
@@ -197,6 +197,26 @@ public function shouldUseNoveltyDetection()
         return $this->getJsonAttr('params.training_data_method') === self::TRAIN_NOVELTY_DETECTION || $this->getJsonAttr('params.training_data_method') === null;
     }
 
+    /**
+     * Determine if this job should Ignore Existing Labels for own/existing Annotation.
+     *
+     * @return bool
+     */
+    public function shouldIgnoreOwnExistingLabel()
+    {
+        return $this->getJsonAttr('params.oa_ignore_existing_label') || $this->getJsonAttr('params.oa_ignore_existing_label', false);
```

Comment:
This should also be enough:
```suggestion
        return $this->getJsonAttr('params.oa_ignore_existing_label', false);
```